en/docs/get-started/tutorials/create-your-first-event-triggered-using-containerized-app.md

```diff
@@ -0,0 +1,125 @@
+# Create an Event-Triggered Integration as a BYOC component
```

Comment:
How about changing the title to "Deploy a Containerized Application as an Event-Triggered Integration"?
I feel the current title does not clearly reflect what we are walking the user through in this tutorial. WDYT?